README.md

```diff
@@ -663,7 +663,7 @@ This will give you a local copy of the branch `daves`, and any update that has b
 ### I want to create a new remote branch from current local one
 
 ```sh
-$ git config --global push.default current
+$ git config push.default current
```

Comment:
I would not change the config for this kind of trivial thing. I'd much rather use 
`git push -u <remote> HEAD`

Reply:
Alright, I will add that part then

Reply:
Done.